tests/http/test_request_normalization.py

```diff
@@ -2,11 +2,12 @@
 import urllib
 
 import six
+from six.moves import urllib_parse as urlparse
 
 import requests
 
 from flex.http import (
-    normalize_request, _tornado_available, _falcon_available, _webob_available
+    normalize_request, _tornado_available, _falcon_available, _webob_available, _django_available
```

Comment:
It is, this was more of a house-cleaning request since the next one will cause it to exceed 100 and then if it's someone who's not PEP8 aware I'll have to ask them to do it :)